ext/drbbdup/drbbdup.dox

```diff
@@ -127,9 +127,9 @@ considers for instrumentation, it also provides a "where" instruction.
 The client must insert code exactly prior to the "where" instruction
 in order to ensure correct instrumentation.
 
-drbbdup also provides drbbdup_is_first_instr() and drbbdup_is_last_instr()
-to determine whether the passed instruction is the first
-or last instruction of a basic block copy respectively.
+drbbdup also provides drbbdup_is_first_instr() drbbdup_is_first_nonlabel_instr()
+and drbbdup_is_last_instr() to determine whether the passed instruction
+is the first or last instruction of a basic block copy respectively.
```

Comment:
I think the docs should also explicitly point out that to pass info from a separate app2app phase the user should store into TLS since it can't use the drmgr user_data out of that phase.

Reply:
Done.